src/main/java/edu/kis/powp/jobs2d/command/DriverCommand.java

```diff
@@ -13,8 +15,14 @@
 	 * 
 	 * @param driver driver.
 	 */
-	public void execute(Job2dDriver driver);
+	void execute(Job2dDriver driver);
+
+	void accept(VisitorCommand visitor);
+
+	Object clone();
+
+	Point getPoint() throws OperationNotSupportedException;
+
+	void setPoint(Point point) throws OperationNotSupportedException;
```

Comment:
I would appreciate an approach that would respect immutability of original command objects.
Not all commands have actual coordinates, so adding getPoint seems a little intrusive. 
There exist many ways to overcome some of those obstacles, e.g., creating new objects (possibly some sub-hierarchy), checking types, using visitors (to avoid casting).